daemon/cluster/cluster.go

```diff
@@ -280,27 +280,30 @@ func (c *Cluster) GetDataPathAddress() string {
 	return ""
 }
 
-// GetRemoteAddress returns a known advertise address of a remote manager if
+// GetRemoteAddressList returns the known advertise address of a remote managers if
 // available.
 // todo: change to array/connect with info
-func (c *Cluster) GetRemoteAddress() string {
+func (c *Cluster) GetRemoteAddressList() []string {
```

Comment:
Yep, personally I always preferred the -List names instead of tons of "s", but won't "fight" for it, if it is the common used approach in the project or go I can take care of the change also on the libnetwork side. let me know